pkg/ocm/helpers.go

```diff
@@ -1106,8 +1235,8 @@ func ValidateClaimValidationRules(input interface{}) error {
 		inputRules = strings.Split(input.(string), ",")
 		for _, inputRule := range inputRules {
 			if !idRE.MatchString(inputRule) {
-				return fmt.Errorf("invalid identifier '%s' for 'claim validation rule. '"+
-					"Should be in a <claim>:<required_value> format.", inputRule)
+				return fmt.Errorf("invalid identifier '%s' for 'claim validation rule' "+
+					"should be in a <claim>:<required_value> format", inputRule)
```

Comment:
Is there a reason for removing the `.` after `'claim validation rule'` and making it into one sentence?